core/src/test/resources/sql/planner.iq

```diff
@@ -0,0 +1,89 @@
+# planner.iq - planner tests can customizable optimization rules
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+!use post
+!set outputformat mysql
+
+# The following line is used to update the default planner rules.
+# "+" means add a rule, "-" means remove a rule. Only the "CoreRules." can be omitted.
+!set planner-rules "-INTERSECT_TO_DISTINCT,-EnumerableRules.ENUMERABLE_INTERSECT_RULE,+CoreRules.INTERSECT_TO_SEMI_JOIN"
```

Comment:
I suspect the quotes are actually not necessary, is that right?
but maybe it's useful to keep them, since we may change Quidem to allow multi-line strings

Reply:
It is still needed at this stage. If there are no double quotes, the reading will be considered to end when a comma is encountered.